src/gameObjects/zombie.js

```diff
@@ -0,0 +1,36 @@
+export default class Zombie {
+    constructor(scene, x, y) {
+      this.scene = scene;
+      this.init(x, y);
+    }
+  
+    init(x, y) {
+      this.sprite = this.scene.matter.add.sprite(0, 0, "zombie", 0);
+  
+      const { Body, Bodies } = Phaser.Physics.Matter.Matter;
+      const { width: w, height: h } = this.sprite;
+      const mainBody = Bodies.rectangle(0, 0, w * 0.75, h *0.75, {
+        chamfer: { radius: 10 },
```

Comment:
**suggestion**: Similar to the Player class, consider making the chamfer radius configurable for consistency and flexibility.